internal/plugins/ansible/v1/api.go

```diff
@@ -141,6 +141,7 @@ func (p *createAPISubcommand) InjectResource(res *resource.Resource) error {
 	}
 
 	// Selected CRD version must match existing CRD versions.
+	// nolint:staticcheck
```

Comment:
Just curious, what is the lint error that we are getting here such that we need to add this?

Reply:
This deprecation message would be from KB (https://github.com/kubernetes-sigs/kubebuilder/pull/3261). It checks if any other version of CRD is tracked in Project config than the one specified while creating the specific resource. We no longer need this since v1beta1 CRDs are not supported.

We could have this for now, since users can still (though not recommended) use a v1beta1 CRD with a plugin and run it against a <k8s1.22 cluster. 

We should add a `//TODO` comment to remove this as we technically don't need this check anymore in future when KB removes this API. We have come a far way along (4 k8s versions higher than when it was removed) to explicitly remove support for v1beta1 CRDs and WHs.